functions/src/updates/deleteUser.js

```diff
@@ -30,11 +30,11 @@ exports.deleteUser = functions
 
         batch.commit();
 
-        const bucket = storage.bucket();
-        bucket.deleteFiles({
-            prefix: 'users/${deletedUserId}/'
+        await storage.bucket('wpa-be-app.appspot.com').deleteFiles({
+            prefix: `responses/${deletedUserId}/`
         });
-        bucket.deleteFiles({
-            prefix: 'responses/${deletedUserId}/'
+        await storage.bucket('wpa-be-app.appspot.com').deleteFiles({
```

Comment:
We should store this `wpa-be-app.appspot.com` as a variable somewhere. this way it's hard to keep track of it or change the env to dev. Ideally we would want these in the config file.